AirCasting/ABConnector/SDSyncController.swift

```diff
@@ -68,13 +69,16 @@ class SDSyncController {
                     self.checkFilesForCorruption(files, expectedMeasurementsCount: metadata.expectedMeasurementsCount) { fileValidationResult in
                         switch fileValidationResult {
                         case .success(let verifiedFiles):
+                            Log.verbose("[TEST] Files verified.")
                             self.handle(files: verifiedFiles, sensorName: sensorName, completion: completion)
                         case .failure(let error):
+                            Log.verbose("[TEST] File verification failed. \(error.localizedDescription)")
                             Log.error(error.localizedDescription)
                             completion(false)
                         }
                     }
                 case .failure:
+                    Log.verbose("[TEST] Failed to download SD card data from AB")
```

Comment:
can we make this into Log.error with proper logging?

Also I think `[TEST]` logs can be removed (including other files). We can review them and leave those that we want to have in logs (switching them to `Log.info`)